src/pages/mypage/components/AcornHistory.tsx

```
@@ -0,0 +1,149 @@
+
+    if (sortKey === "recent") {
+      copied.sort((a, b) => b.date.localeCompare(a.date));
+    }
+    if (sortKey === "oldest") {
+      copied.sort((a, b) => a.date.localeCompare(b.date));
+    }
+
+    return copied;
+  }, [items, filter, sortKey]);
+
+  useEffect(() => {
```

Biome (2.3.13)

[error] This hook specifies more dependencies than necessary: filtered.length, loadSize.

This dependency can be removed from the list.

This dependency can be removed from the list.

React relies on hook dependencies to determine when to re-compute Effects.
Specifying more dependencies than required can lead to unnecessary re-rendering
and degraded performance.
Unsafe fix: Remove the extra dependencies from the list.


(lint/correctness/useExhaustiveDependencies)